### PR TITLE
fix: add zh-cn community

### PR DESCRIPTION
**Pull request by LaoshuBaby**
### Description

Add community link of zh-cn localization team from China

### Motivation

Many times translator complaining that they can't find a discussion forum in Simplified Chinese in Taiwan's group

### Additional details

None.

### CODE REVIEW

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -21,8 +21,9 @@ original_slug: MDN/Contribute/Localize
 
 ### 汉语（zh-CN，zh-TW）
 
-- 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)
-- 目前的志愿者：[Irvin](https://github.com/irvin)、[t7yang](https://github.com/t7yang)、[yin1999](https://github.com/yin1999)
+- Moz://TW 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)
+- Mozilla China 讨论组：[Mozilla China](https://t.me/mozilla_china)
+- 目前的志愿者：[Irvin](https://github.com/irvin)、[t7yang](https://github.com/t7yang)、[yin1999](https://github.com/yin1999)、[LaoshuBaby](https://github.com/LaoshuBaby)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:26]: This is used to list l10n team members, so please revert this change

image

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 2]
```diff
@@ -21,8 +21,9 @@ original_slug: MDN/Contribute/Localize
 
 ### 汉语（zh-CN，zh-TW）
 
-- 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)
-- 目前的志愿者：[Irvin](https://github.com/irvin)、[t7yang](https://github.com/t7yang)、[yin1999](https://github.com/yin1999)
+- Moz://TW 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)
+- Mozilla China 讨论组：[Mozilla China](https://t.me/mozilla_china)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:25]:
```suggestion
- 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/tg)、[Mozilla China](https://t.me/mozilla_china)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/mdn/community/contributing/translated_content/index.md (hunk 1 above)
Comment (LaoshuBaby) [reply files/zh-cn/mdn/community/contributing/translated_content/index.md:26]: Thanks for pointing out that if there is a clear source for this, it must be according to the source. 
I added myself before because I want to help establish communication when noticed someone said that they can't get in touch with others.

**Review by yin1999 — APPROVED** [review]
Please also create a PR in [mdn/content](https://github.com/mdn/content)

**Comment by yin1999** [comment]
Merged in upstream, let's do this in translated-content.

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ original_slug: MDN/Contribute/Localize
 
 ### 汉语（zh-CN，zh-TW）
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:22]:
```suggestion
### 汉语（zh-CN、zh-TW）
```
